ChocoCCM.build.ps1

```diff
@@ -0,0 +1,286 @@
+[CmdletBinding()]
+param(
+    [Parameter()]
+    [string]
+    $OutputDirectory = "$PSScriptRoot/Output",
+
+    [Parameter()]
+    [string]
+    $TimeStampServer = $(
+        if ($env:CERT_TIMESTAMP_URL) {
+            $env:CERT_TIMESTAMP_URL
+        }
+        else {
+            'http://timestamp.digicert.com'
+        }
+    ),
+
+    [string]
+    $CertificatePath = $env:CHOCOLATEY_OFFICIAL_CERT,
+
+    [string]
+    $CertificatePassword = $env:CHOCOLATEY_OFFICIAL_CERT_PASSWORD,
+
+    [Parameter()]
+    [string]
+    $CertificateAlgorithm = $(
+        if ($env:CERT_ALGORITHM) {
+            $env:CERT_ALGORITHM
+        }
+        else {
+            'Sha256'
+        }
+    ),
+
+    [string]
+    $CertificateSubjectName = "Chocolatey Software, Inc.",
+
+    [Parameter()]
+    [string]
+    $NugetApiKey = $env:POWERSHELLPUSH_API_KEY,
+
+    [Parameter()]
+    [string]
+    $PublishUrl = $env:POWERSHELLPUSH_SOURCE
+)
+
+$script:SourceFolder = "$PSScriptRoot/src"
+$script:ReleaseBuild = [string]::IsNullOrEmpty((git tag --points-at HEAD 2> $null) -replace '^v')
+
+# Synopsis: install Dotnet CLI
+task InstallDotNet {
+    $DotNetChannel = "Current"
+    $DotNetVersion = "latest"
+    $DotNetInstallerUri = "https://dot.net/v1/dotnet-install.ps1"
+
+    function Remove-PathVariable([string]$VariableToRemove) {
+        $path = [Environment]::GetEnvironmentVariable("PATH", "User")
+        if ($path) {
+            $newItems = $path.Split(';', [StringSplitOptions]::RemoveEmptyEntries) | Where-Object { "$($_)" -notlike $VariableToRemove }
+            [Environment]::SetEnvironmentVariable("PATH", [System.String]::Join(';', $newItems), "User")
+        }
+
+        $path = [Environment]::GetEnvironmentVariable("PATH", "Process")
+        if ($path) {
+            $newItems = $path.Split(';', [StringSplitOptions]::RemoveEmptyEntries) | Where-Object { "$($_)" -notlike $VariableToRemove }
+            [Environment]::SetEnvironmentVariable("PATH", [System.String]::Join(';', $newItems), "Process")
+        }
+    }
+
+    # Get .NET Core CLI path if installed.
+    $FoundDotNetCliVersion = $null
+    if (Get-Command dotnet -ErrorAction SilentlyContinue) {
+        $FoundDotNetCliVersion = dotnet --version
+    }
+
+    if ($FoundDotNetCliVersion -ne $DotNetVersion) {
```

Comment:
Possibly? I stole this code verbatim from Stevie's original build script.

But tbh I've gone and removed this installation of dotnet entirely. We were only using it to install the gitversion tool, which doesn't even work with the latest version of dotnet (it complains that the version of dotnet is too new). Assuming it only worked locally because I have a zillion dotnet versions installed.

I've replaced this section with a `choco install gitversion.portable` for the time being.

Reply:
What version of GitVersion were you trying to use. We are currently successfully using this version in Chocolatey.Cake.Recipe: https://github.com/chocolatey/chocolatey.cake.recipe/blob/develop/Chocolatey.Cake.Recipe/Content/toolsettings.cake#L33